src/commands/certify.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
this probably won't be built by @pika/pack (our package builder)

Instead, put this file into `assets/certify.sh`, so that it will be copied over to the final `pkg` directory to `pkg/assets`.